mozilla/blurts-server
# MNTOR-2136: fix filtering changes dashboard top banner

# References: 
Jira: MNTOR-2136



# Description
We had a conditional in a couple of places where it assumes "exposure card list is empty" equates "all fixed".
When we filter, the exposure card list changes, and if it's changed to [], then the app assumes "all fixed"
Here we added a new variable that actually counts from summary the "all fixed" state and pass that into the top banner component.

# How to test
* Navigate to the Monitor redesigned dashboard
* Filter the exposures by data breaches and observe the overview card message
* Filter the exposures by info for sale and observe the overview card message;
* Expected:
  * The overview card displays the same message, reflecting the actual state of the user’s exposures

# Checklist (Definition of Done)
